src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -59,6 +59,11 @@ protected override sealed Symbol OriginalSymbolDefinition
         /// </summary>
         public abstract RefKind RefKind { get; }
 
+        /// <summary>
+        /// Returns true if the parameter is a discard parameter.
+        /// </summary>
+        public abstract bool IsDiscard { get; }
```

Comment:
>public abstract bool IsDiscard { get; } [](start = 8, length = 39)

Consider adding an explicit implementation for ```IParameterSymbol.IsDiscard```. This will make it easier to deal with a merge conflict with the change that splits ISymbols and Symbols for C#.